### ISSUE TITLE
Support, or document the support for the --quiet flag (to suppress warnings)

### ISSUE THREAD

**Issue by oveddan** [adametry/gulp-eslint#95]
Is it possible to pass [--quiet](http://eslint.org/docs/user-guide/command-line-interface#quiet) as an option to gulp-eslint?
If not, this would be useful to suppress warning.
If so, some documentation should be added for this flag.

**Comment by adametry** [comment]
There will be soon. A release candidate has been published to gather feedback on several updates, including a "quiet" option that can be set to `true` (=remove warning message) or a `function(message)` that returns a true/false value for whether to include the message or not. For example:
```javascript
gulp.task('lint-quiet', function () {
	return gulp.src('src/**/*.js')
		.pipe(eslint({quiet: true}))
		.pipe(eslint.format());
});
```
...or *only* warnings:
```javascript
function isWarning(message) {
	return message.severity === 1;
}
gulp.task('lint-warn', function () {
	return gulp.src('src/**/*.js')
		.pipe(eslint({quiet: isWarning}))
		.pipe(eslint.format());
});
```
Try the release candidate out and let me know how well it works for you.
```javascript
npm install gulp-eslint@rc
```

**Comment by adametry** [comment]
gulp-eslint@1.1.0 has been published and the README/doc has been updated. I'll close this issue since it has been addressed. Let me know if any problems remain.
